index.js

```diff
@@ -3,53 +3,198 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
-    }
+      price: 0.35,
+    },
   ],
-  cart: []
-};
\ No newline at end of file
+  cart: [],
+};
+const shop = document.querySelector(".store--item-list");
+const cart = document.querySelector(".cart--item-list");
+const totalNumber = document.querySelector(".total-number");
+
+function renderShop(){
+  shop.innerHTML = ""
+  state.items.forEach((item) => {
+    // console.log("eachOne", item)
+    const li = document.createElement("li");
+    const div = document.createElement("div");
+    div.setAttribute("class", "store--item-icon");
+    const img = document.createElement("img");
+    img.src = `assets/icons/${item.id}.svg`;
+    img.alt = `${item.name}`;
+    const addButton = document.createElement("button");
+    addButton.innerText = "add to cart";
+    addButton.addEventListener("click", function (e) {
+      // e.preventDefault();
+      // state.cart.push(item)
+      console.log("here is the id", item.id);
+      // if (cartItem.id === item.id){
+      //   // quantity =+1
+    //   return true;
+    //   // console.log("Quantity", quantity)
+    // }else{
+    //   return false
+    // state.cart.push(item)
+    // }
+    
+    if (!item.quantity) {
```

Comment:
What does this section from 85 to 90 do?

Reply:
I am also interrested in this section.